src/Hex.jsx

```diff
@@ -147,7 +147,7 @@ const HexTile = ({ hex, id, mask, border, transparent, map }) => {
   );
   let mediumCityBorders = (
     <Position data={hex.mediumCities}>
-      {m => <MediumCity border={true} />}
+      {m => <MediumCity border={true} {...m} />}
```

Comment:
In this case the `{...m}` should be before the `border={true}` so that IF someone did say `"border": false` in their definition it gets overridden. Unlikely case... but.